sharding/multiShardCoordinator.go

```diff
@@ -45,16 +45,21 @@ func (msc *multiShardCoordinator) calculateMasks() (uint32, uint32) {
 	return (1 << uint(n)) - 1, (1 << uint(n-1)) - 1
 }
 
-// ComputeId calculates the shard for a given address used for transaction dispatching
+// ComputeId calculates the shard for a given address container
 func (msc *multiShardCoordinator) ComputeId(address state.AddressContainer) uint32 {
+	return msc.ComputeIdFromBytes(address.Bytes())
+}
+
+// ComputeIdFromBytes calculates the shard for a given address
+func (msc *multiShardCoordinator) ComputeIdFromBytes(address []byte) uint32 {
```

Comment:
you did it :D

Reply:
I just prepared the method, is not yet added in interface and mocks. It should be done on other PR